src/wasm.rs

```diff
@@ -971,7 +989,12 @@ impl WasmModule {
                             mutable: global_ty.mutable,
                         });
                     }
-                    e => panic!("Have not implemented import section entry type {:?}", e),
+                    ImportSectionEntryType::Memory(memory_ty) => {
+                        self.memories.push(*memory_ty);
+                    }
+                    ImportSectionEntryType::Table(table_ty) => {
+                        self.tables.push(*table_ty);
+                    }
```

Comment:
Does `aWsm` import tables and memories?

Reply:
So! It does if you incrementally compile.

```
a.c --> a.wasm.o -.
b.c --> b.wasm.o -+-> combined.wasm
c.c --> c.wasm.o -'
            ^- These all used table/memory imports
```

It doesn't come up for the code_benches, but was a problem when I was applying aWsm to an existing project+build system.

Reply:
Second pass note: This seems to only happen if you incrementally compile _without_ LTO. With LTO the intermediaries are just LLVM IR blobs.

So it's uncommon, but I have locally tested this implementation so I figured it was worth upstreaming.